packages/es-components/src/components/containers/prompt/Prompt.tsx

```diff
@@ -33,14 +42,25 @@ const Content = styled.div`
   word-break: break-word;
 `;
 
-const BannerLabel = styled(Label)`
+interface BannerLabelProps {
+  bannerTextColor: string;
+}
+
+const BannerLabel = styled(Label)<BannerLabelProps>`
   letter-spacing: 0.5px;
   font-size: 20px;
   font-weight: 500;
   color: ${promptProps => promptProps.bannerTextColor};
 `;
 
-const Prompt = props => {
+interface PromptProps {
```

Comment:
Just for downstream convenience:
```suggestion
export interface PromptProps {
```